src/edgepi/adc/edgepi_adc.py

```diff
@@ -131,52 +128,74 @@ def __get_current_rtd_state(self) -> dict[str, PropertyValue]:
         return {
             "adc_1_analog_in": self.adc_1.mux_p.code,
             "adc_1_mux_n": self.adc_1.mux_n.code,
+            "adc_2_analog_in": self.adc_2.mux_p.code,
+            "adc_2_mux_n": self.adc_2.mux_n.code,
             "idac_1_mux": self.__get_state(ADCProperties.IDAC1_MUX).code,
             "idac_2_mux": self.__get_state(ADCProperties.IDAC2_MUX).code,
             "idac_1_mag": self.__get_state(ADCProperties.IDAC1_MAG).code,
             "idac_2_mag": self.__get_state(ADCProperties.IDAC2_MAG).code,
             "pos_ref_inp": self.__get_state(ADCProperties.REFMUX_POS).code,
             "neg_ref_inp": self.__get_state(ADCProperties.REFMUX_NEG).code,
+            "adc2_ref_inp": self.__get_state(ADCProperties.ADC2_REFMUX).code,
         }
 
-    def __get_rtd_state(self) -> bool:
+    def __get_rtd_state(self):
         """
         Get on/off RTD state.
+        Returns:
+            RTD
         """
```

Comment:
first make sure to fix the doc string. can you define what we mean by rtd state? is it a combination of rtd mode and rtd adc ? if so why not create a proper enum like `RTDState` and define four clear values as four tuples like:
```
class RTDState(enum):
    RTD_ON_ADC1 = (RTDModes.RTD_ON, ADC1RtdConfig.ON),
    RTD_OFF_ADC1 = (RTDModes.RTD_OFF, ADC1RtdConfig.OFF),
    RTD_ON_ADC2 = (RTDModes.RTD_ON, ADC2RtdConfig.ON),
    RTD_OFF_ADC2 = (RTDModes.RTD_OFF, ADC2RtdConfig.OFF)
```

Reply:
Ah, I think this should've been a topic from yesterday's meeting. I think RTD state should be a combination of both RTDMode and RTDADC.

If the user request to get information regarding RTD, the SDK should report the state of RTD configuration.